doc/man3/OSSL_CRMF_MSG_set1_regCtrl_regToken.pod

```diff
@@ -28,20 +37,32 @@ OSSL_CRMF_CERTID_gen
                                       int method, GENERAL_NAME *nm);
  int OSSL_CRMF_MSG_set_PKIPublicationInfo_action(
                                   OSSL_CRMF_PKIPUBLICATIONINFO *pi, int action);
+ const OSSL_CRMF_PKIPUBLICATIONINFO
+     *OSSL_CRMF_MSG_get0_regCtrl_pkiPublicationInfo(const OSSL_CRMF_MSG *msg);
  int OSSL_CRMF_MSG_set1_regCtrl_pkiPublicationInfo(OSSL_CRMF_MSG *msg,
                                         const OSSL_CRMF_PKIPUBLICATIONINFO *pi);
+ const X509_PUBKEY
+     *OSSL_CRMF_MSG_get0_regCtrl_protocolEncrKey(const OSSL_CRMF_MSG *msg);
  int OSSL_CRMF_MSG_set1_regCtrl_protocolEncrKey(OSSL_CRMF_MSG *msg,
                                                 const X509_PUBKEY *pubkey);
+ const OSSL_CRMF_CERTID
+     *OSSL_CRMF_MSG_get0_regCtrl_oldCertID(const OSSL_CRMF_MSG *msg);
  int OSSL_CRMF_MSG_set1_regCtrl_oldCertID(OSSL_CRMF_MSG *msg,
                                           const OSSL_CRMF_CERTID *cid);
  OSSL_CRMF_CERTID *OSSL_CRMF_CERTID_gen(const X509_NAME *issuer,
                                         const ASN1_INTEGER *serial);
 
 =head1 DESCRIPTION
 
+OSSL_CRMF_MSG_get0_regCtrl_regToken() returns the regToken control
+in the given I<msg>, if present.
+
 OSSL_CRMF_MSG_set1_regCtrl_regToken() sets the regToken control in the given
 I<msg> copying the given I<tok> as value. See RFC 4211, section 6.1.
 
+OSSL_CRMF_MSG_get0_regCtrl_regToken() returns the authenticator control
```

Comment:
nit: `OSSL_CRMF_MSG_get0_regCtrl_regToken` is mentioned 4 times.

Reply:
Oops, this was a copy&paste mistake I did in the rush. 
Fixed by making the text generic, which is more succinct and should be better to read.

Reply:
Thank you @FdaSilvaYY for having an eye on glitches in the documentation in may new PRs.